src/Visualization/Python/Render1D.py

```diff
@@ -0,0 +1,277 @@
+
+    else:
+        for file_name in filename_list:
+            try:
+                h5_files.append(h5py.File(file_name, 'r'))
+            except OSError:
+                exit("Could not find file %s in directory %s" %
+                     (file_name, os.path.dirname(file_name)))
+    return h5_files
+
+
+def print_var_names(args):
+    '''
+    Print all available variables to screen
+    :param args:
+    :return: None
+    '''
+
+    file_prefix = args["file_prefix"]
+    filename_list = args["filename_list"]
+    h5files = get_h5_files(file_prefix, filename_list)
+    volfile = h5files[0]["element_data.vol"]
+    obs_id_0 = next(iter(volfile))
+    variables = list(volfile[obs_id_0].keys())
+    variables.remove("connectivity")
+    variables.remove("InertialCoordinates_x")
+    variables.remove("total_extents")
+    variables.remove("grid_names")
+    print("Variables in H5 file:\n[%s]" % ', '.join(map(str, variables)))
+    for h5_file in h5files:
+        h5_file.close()
+    return None
+
+
+def get_data(args):
+    '''
+    Get the data to be plotted
+    :return: the array of data, coords, and time
+    '''
+    time = []
+    coords = []
+    data = []
+    file_prefix = args["file_prefix"]
+    filename_list = args["filename_list"]
+    var_name = args["var"]
+    h5files = get_h5_files(file_prefix, filename_list)
+    volfiles = [h5file['element_data.vol'] for h5file in h5files]
+    # Get a list of times from the first vol file
+    ids_times = [(obs_id, volfiles[0][obs_id].attrs['observation_value'])
+                 for obs_id in volfiles[0].keys()]
+    ids_times.sort(key=lambda pair: pair[1])
+    time_data = {}
+    for id_and_time in ids_times:
+        time_data[id_and_time] = []
+        local_data = []
+        local_coords = []
+        for volfile in volfiles:
+            obs_id_file = volfile[id_and_time[0]]
+            local_data += list(obs_id_file[var_name])
+            local_coords += list(obs_id_file['InertialCoordinates_x'])
+        local_coords_and_data = zip(local_coords, local_data)
+        time_data[id_and_time] += local_coords_and_data
+
+        time_data[id_and_time].sort(key=lambda pair: pair[0])
+        time.append(id_and_time[1])
+        coords.append([pair[0] for pair in time_data[id_and_time]])
+        data.append([pair[1] for pair in time_data[id_and_time]])
+
+    for h5file in h5files:
+        h5file.close()
+    return time, coords, data
+
+
+def render_single_time(args, coords, data):
+    '''
+    Renders a single time snap shot
+    :param args: the command line arguments
+    :return: None
+    '''
+    plt.xlabel("x")
+    plt.ylabel(args['var'])
+    plt.title("t = %.5f" % int(args['time']))
+    plt.plot(coords[int(args['time'])], data[int(args['time'])], 'o')
+    if args['save']:
+        print("Writing still to file %s.pdf" % args['save'])
+        plt.savefig("%s.pdf" % args['save'],
+                    transparent=True,
```

Comment:
Why do make the background transparent? We can't include all possible styling possibilities in this script, so if a person wants to plot the data and then style it differently they can import and call these functions and add their styling code in their script. By default this should probably not be transparent, and neither use "tight layout".